src/PostgREST/Error.hs

```diff
@@ -37,12 +39,14 @@ import qualified PostgREST.MediaType        as MediaType
 
 import PostgREST.SchemaCache.Identifiers  (QualifiedIdentifier (..))
 import PostgREST.SchemaCache.Proc         (ProcDescription (..),
-                                           ProcParam (..))
+                                           ProcParam (..), ProcsMap)
 import PostgREST.SchemaCache.Relationship (Cardinality (..),
                                            Junction (..),
                                            Relationship (..))
 import Protolude
 
+-- >>>
+-- >>>
```

Comment:
Hm, why is this needed? Can it be deleted?

Reply:
Oops, forgot to remove those.